OpenFEC-Data-Visualization/blog.md

```diff
@@ -0,0 +1,247 @@
+# Introduction to OpenFEC Data Visualization with ChartJs
+## Purpose
+
+The importance of understanding politics and the implications of being informed voters are profound. The ability for citizens of a country to understand the financial spending of candidates running for office (at any level) allows one to understand where a candidate's priorities lie. Observing and analyzing financial trends allows one to make predictions about who may win a particular election. A lot of the time, however, it’s easy to get lost in the numbers! 
+
+Thus, being able to visualize this data in a graphical manner helps in truly comparing and understanding a candidate’s financial spending in comparison to others in an easy manner. However, how do you verify the graphs you see online are indeed accurate? 
+
+It’s a catch-22 situation! Either you receive accurate numbers (hard to analyze) or potentially biased graphs. Today, you’ll learn how to make the graph yourself, using the official financial data available at the OpenFEC API.
+
+For this tutorial, we’ll walk through how to make a bar graph visualizing various financial information (operating expenses, net receipts, fundraising disbursements, and individual contribution less refunds) for Democrats and Republicans as a whole, as well as specific candidates for the 2016 presidential election.
+
+## Setup
+
+1. Sign up for the OpenFEC API Key through this [website.](https://api.data.gov/signup/)
+
+2. Check your email for the API Key and save it
+
+3. Download ChartJS through this [website.](https://www.jsdelivr.com/package/npm/chart.js) This is done by clicking the download icon (fourth button), which will download a zip file. Once the zip file is downloaded, move to the desired location where you will store all your project files.
+
+   <img src = "./pictures/chartjs-download.png"/>
+
+4. Create one HTML file (call it OpenFEC.html) and one JS file (call it OpenFEC.js) in the same location as ChartJS files you downloaded in step 3.
+
+5. Download a text editor if you don’t have one. You'll need one to code everything up!
+
+6. At the end, to see your output, right-click on the HTML file and open with Google Chrome!
+
+## How Will We Use the API?
+
+Now that you have the ChartJS dependency, HTML (OpenFEC.html) and JS (OpenFEC.js) files set up, let’s get to coding!
+
+In OpenFEC.html, add the following lines of code.
+
+```html
+<!-- Create a canvas object for ChartJS -->
+<canvas id="chart" width="400" height="400"></canvas>
```

Comment:
Instead of having a code block with giant text description, split it up! 

```html
<!-- Create a canvas object for ChartJS -->
<canvas id="chart" width="400" height="400"></canvas>
```
We use the canvas tag (needed for chartJS) with the specified width and height to place our graph. This is how our graph will go on the webpage!